WHATWG_SPEC_LICENSE

```diff
@@ -0,0 +1,13 @@
+Copyright 2025 WHATWG (Apple, Google, Mozilla, Microsoft)
```

Comment:
That's the copyright of WHATWG specs, but not sure if we should put that here directly. Firefox at least doesn't do that: https://searchfox.org/mozilla-central/source/toolkit/content/license.html#2275

As first line I'd put:
```suggestion
BSD 3-Clause License
```

Reply:
I don't think that is correct, because:

- FF has this [`See the individual LICENSE files for copyright owners.`](https://searchfox.org/mozilla-central/rev/e339b9d6175e30310f6891a5e85d27ce43ad94f1/toolkit/content/license.html#2286), and links to specific folder, that contain a BSD 3 license with copyright owners, for example [`REACT_TRANSITION_GROUP_LICENSE`](https://searchfox.org/mozilla-central/rev/e339b9d6175e30310f6891a5e85d27ce43ad94f1/toolkit/content/vendor/react/REACT_TRANSITION_GROUP_LICENSE).
- It follows the license template at https://opensource.org/license/BSD-3-Clause

It seems FF forgot to add a specific file for the specs. It seems to me that if you add a license it should contain the copyright owners.

Reply:
I'm not really sure, we don't have any copyright header in our main `LICENSE` file either which is strange, anyway I think it's fine if you prefer to keep the Copyright line, I'd also add the title of the LICENSE on the top, similar to what we do in the `LICENSE` file.

Reply:
Ok